docs/docs/reference/functions/getting-started.md

```diff
@@ -190,6 +190,16 @@ export default function FormPage() {
 }
 ```
 
+## Functions in plugins and themes
```

Comment:
A general note: there's inconsistency for how we talk about functions. Gatsby Functions? Serverless functions? Functions or functions (capitalization)?

We should pick one way and be consistent. (I'll open a thread on it in Slack.)

Reply:
hmm yeah. I thought about it and decided that when talking about functions in general, I'd use lower-case `function` and only use upper case when talking about the feature specifically. Using upper-case all the time feels fussy.